fluentd/source.jemalloc

```diff
@@ -1,3 +1,3 @@
-JEMALLOC_VER=5.2.0
-JEMALLOC_SOURCE=https://github.com/jemalloc/jemalloc/releases/download/${JEMALLOC_VER}/jemalloc-${JEMALLOC_VER}.tar.bz2
-JEMALLOC_SHA512SUM=e3be4d534770126caf10f2684aed9fe4ba1422dd47625fe50343cfb750f26eff869fcc7d1e30a96dd6c73f6614c4bbcd560fd24fc26b55ac731c43e60fd05234
+JEMALLOC_VER=5.2.1
```

Comment:
`JEMALLOC_VER` is unused?

Reply:
Good catch I can fix this in a follow-up PR no deal breaker for now.